## Hi there ğŸ‘‹

# ğŸ‘©â€ğŸ”¬ Mohini Jaiswal, Ph.D.

**Computational Biologist | Genomics Researcher | ML & Protein Language Models Enthusiast**

ğŸ”¬ Passionate about harnessing machine learning and AI-based approach to deep dive and decode biological data to drive therapeutic discovery.

ğŸ“ Based in Boston, MA | ğŸ“§ jaiswalmohini189@gmail.com | ğŸŒ [LinkedIn](https://www.linkedin.com/in/mohini-jaiswal-705073366)

---

## ğŸ‘‹ About Me

class about_me:

    def __init__(self, name, role):
        self.name = "Mohini Jaiswal, PhD"
        self.role = "Computational Biologist"
    
    def research_experience(self):
        self.PhD = "Plant Genome Research"
        self.Postdoc = "Antimicrobial Drug Discovery"

    def research_interests(self):
        domain = ['Functional genomics','multi-omics integration',
                  'Protein structure','peptide prediction','AI/ML-driven drug discovery',
                  'Metagenomics & biosynthetic gene clusters','Plant-derived Bioactive peptides']

Intro = about_me("name", "role")

Intro.research_experience()

print(f"I'm a {Intro.role} with a Ph.D. in {Intro.PhD} and postdoctoral experience in {Intro.Postdoc}.")

##
* My work sits at the intersection of **genomics**, **machine learning**, and **bioinformatics**, where I develop scalable pipelines, predictive models, and tools to accelerate scientific research.
* Currently a Postdoctoral Research Associate at the **Antimicrobial Discovery Center, Northeastern University**, where I work with NGS data, protein language models, and machine learning pipelines for biosynthetic gene cluster and target prediction and metagenomic analysis.

---

## ğŸ› ï¸ Technical Skills

**Languages & Tools**  
Python, R, Bash, HTML/CSS/JS, MySQL, QIIME, antiSMASH, Kraken, SignalP, InterProScan, SWISS-MODEL, DESeq2, Bowtie, BLAST, bedtools, samtools  

**Machine Learning & Deep Learning**  
Scikit-learn, PyTorch, WEKA  
SVM, Random Forest, ANN, Naive Bayes, KNN  
Feature selection (RFE, PCA), Cross-validation  
Protein Language Models: ESM, ProtBERT, AlphaFold2

**Visualization**  
Matplotlib, Seaborn, Plotly, ggplot2  

---

## ğŸ“š Publications

- **smAMPsTK** â€“ A toolkit for decoding smORFome encoding AMPs in plants  
  *J Biomol Struct Dyn, 2023* [PMID: 37464885]

- **PTPAMP** â€“ A tool for predicting plant-derived antimicrobial peptides  
  *Amino Acids, 2022* [PMID: 35864258] â€“ *Presented at ISMB 2022*

- **PlantPepDB** â€“ A curated plant peptide database  
  *Scientific Reports, 2020* [PMID: 32042035]

---

## ğŸŒ Conferences & Workshops

- ECML-PKDD 2024 (Vilnius, Lithuania)  
- ISMB 2022 (Poster, Madison, WI)  
- National Conf. on Plant Biology (CSIR-IIIM, 2022)  
- NGS & Ensembl Workshop (NIPGR, 2019)

---

## ğŸ“¬ Get in Touch

If you're interested in collaboration, research goals, or just want to chat about AI in biology â€” feel free to reach out!

ğŸ“§ jaiswalmohini189@gmail.com  
ğŸ”— [LinkedIn](https://www.linkedin.com/in/mohini-jaiswal-705073366)  
ğŸ™ [GitHub](https://github.com/JAIS-95)

---
